src/malay/LexiconMay.gf

```diff
@@ -320,7 +323,8 @@ lin sell_V3 = mkV3 (mkV "jual" Meng) emptyPrep emptyPrep ; -- TODO
 -- lin shop_N = mkN "" ;
 -- lin short_A = mkA "" ;
 -- lin silver_N = mkN "" ;
--- lin sing_V = mkV "" ;
+lin sing_V = mkV "nyanyi" ;
+lin sing_V2 = mkV2 "nyanyi" ;
```

Comment:
Also don't commit the sing_V2 here.

If you want to have your custom lexicon for testing purposes, you could add stuff into `AllMayAbs` and `AllMay`, because the `All*` modules don't share an abstract syntax. And since #387 , `All*` modules can now be used in unit tests too.